src/Networking/client.cpp

```diff
@@ -74,6 +72,58 @@ int Client::sendRequest(const char* data) {
         return 1;
     }
     cout << "Client Received: " << string(buffer, mLen) << endl;
+
+    /*
+     * If the request received contains the keyword "get", which is used to
+     * represent a file transfer request, the client will proceed to provide the
+     * file.
+     */
+    char *token, *dummy;
+    dummy = buffer;
+    token = strtok(dummy, " ");
+    // process file descriptor
+    if (strcmp("get", token) == 0) {
```

Comment:
would this comparison change when we add the protobuf serialization?

Reply:
It wouldn't. This is part of the system command that tells the "FTP server" it wants a file (ie. `get trainingData.proto`). We will have to change everything that marshals from a file into `vector<int>` though. 

```
/*
     * If the request received contains the keyword "get", which is used to
     * represent a file transfer request, the client will proceed to provide the
     * file.
     */
     
```